### PR TITLE
install.sh: convert relocate_python_scripts.py to a bash function

### PR DESCRIPTION
**Pull request by syuu1228**
Since we need to run relocate_python_scripts.py on install time,
python script may not able to run on various different environment.
So convert the script to bash script, merge it into install.sh.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -103,6 +103,31 @@ EOF
     chmod +x "$root/$prefix/bin/$bin"
 }
 
+relocate_python3() {
+    local script="$1"
+    local scriptdir="$(dirname $script)"
```
Comment (avikivity) [inline install.sh]: for extra safety, "$(dirname "$script")"

File: install.sh [hunk 2]
```diff
@@ -103,6 +103,31 @@ EOF
...
+    local relocated="$scriptdir/libexec/$(basename $script)"
```
Comment (avikivity) [inline install.sh]: Here too

File: install.sh [hunk 3]
```diff
@@ -103,6 +103,31 @@ EOF
...
+    local pythonpath="$(dirname $(realpath -ms --relative-to $scriptdir $rpython3))"
```
Comment (avikivity) [inline install.sh]: and here

File: install.sh [hunk 4]
```diff
@@ -103,6 +103,31 @@ EOF
...
+
+    mv $script $relocated
```
Comment (avikivity) [inline install.sh]: and everywhere

File: install.sh [hunk 5]
```diff
@@ -270,20 +298,13 @@ EOS
     ln -srf "$retc"/systemd/system/node-exporter.service.d/nonroot.conf ~/.config/systemd/user/node-exporter.service.d
 fi
 
-install -m755 scylla-gdb.py -Dt "$rprefix"/scripts/
+install -m644 scylla-gdb.py -Dt "$rprefix"/scripts/
 
-PYSCRIPTS=$(find dist/common/scripts/ -maxdepth 1 -type f -exec grep -Pls '\A#!/usr/bin/env python3' {} +)
+PYSCRIPTS=$(find "$rprefix"/scripts/ -maxdepth 1 -type f -executable -exec grep -Pls '\A#!/usr/bin/env python3' {} +)
```
Comment (avikivity) [inline install.sh]: These changes are good, but are unrelated

File: install.sh (hunk 1 above)
Comment (syuu1228) [reply install.sh]: fixed, force pushed.

File: install.sh (hunk 5 above)
Comment (syuu1228) [reply install.sh]: Dropped these changes for now, adjust relocate_python3 function to do same behavior with relocate_python_script.py

**Comment by syuu1228** [comment]
@avikivity ping

**Pull request merged**